scripts/aws/setupdb.sh

```diff
@@ -69,7 +69,10 @@ envdir /etc/mmw.d/env /opt/app/manage.py migrate
 
 function download_and_load {
     for f in "${FILES[@]}"; do
-        curl -s $FILE_HOST/$f | gunzip -q | psql --single-transaction
+        rm -f tmp.sql tmp.sql.gz
```

Comment:
Any reason to prefer storing a file rather than piping?

Also, does this mean that the last call to `download_and_load` will leave a `tmp.sql` file in the directory?